src/main/kotlin/racingCar/domain/Cars.kt

```diff
@@ -1,19 +1,20 @@
 package racingCar.domain
 
-class Cars(carNumber: Int, private val strategy: MoveStrategy) {
+class Cars(usernames: List<Username>, private val strategy: MoveStrategy) {
 
-    val cars: List<Car> = List(carNumber) { Car() }
+    val cars: Map<Username, Car> = usernames.map { it to Car() }.toMap()
```

Comment:
코틀린 Collection API 의 `associateWith()`를 사용하면 좀 더 간결하게 표현 가능해요~
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/associate-with.html
```suggestion
    val cars: Map<Username, Car> = usernames.associateWith { Car() }
```